### PR TITLE
rKibrantn/blob sample default azure credential 3

### PR DESCRIPTION
**Pull request by rakshith91**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/storage/azure-storage-blob/samples/blob_samples_authentication.py [hunk 1]
```diff
@@ -106,3 +107,21 @@ def auth_shared_access_signature(self):
...
+        from azure.identity import DefaultAzureCredential
+        default_credential = DefaultAzureCredential()
+ 
+        # Instantiate a BlobServiceClient using a token credential
+        from azure.storage.blob import BlobServiceClient
+        blob_service_client = BlobServiceClient(account_url=self.oauth_url, credential=default_credential)
+        # [END create_blob_service_client_oauth]
+ 
+        # Get account information for the Blob Service
+        account_info = blob_service_client.get_service_properties()
```
Comment (KieranBrantnerMagee) [inline sdk/storage/azure-storage-blob/samples/blob_samples_authentication.py]: Trailing newline may break pylint, if not I wouldn't worry about it.

**Comment by KieranBrantnerMagee** [comment]
Thanks again for all your help in detangling the git history/conflict mess.  IOU coffee++

File: sdk/storage/azure-storage-blob/samples/blob_samples_authentication.py (hunk 1 above)
Comment (rakshith91) [reply sdk/storage/azure-storage-blob/samples/blob_samples_authentication.py]: fixed it :)

**Pull request merged**